<launch>
  <node pkg="tf" type="static_transform_publisher" name="base_frame_laser" args="0 0 0 0.0 0 0 /base_link /laser 10" />
  <node pkg="tf" type="static_transform_publisher" name="map_to_odom" args="5.0 0 0 0.0 0 0 /map /odom 2" />
  
  <node pkg="nav_sim" type = "sim_bot.py" name = "sim_bot" />

  <!-- Run the map server -->
  <arg name="map_file" default="$(find nav_sim)/config/yard_map_res_0p5m.yaml"/>
  <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" />
</launch>
